server/form-pages/assess/assessApplication/suitablityAssessment/applicationTimeliness.ts

```diff
@@ -64,9 +64,17 @@ export default class ApplicationTimeliness implements TasklistPage {
     )
     const arrivalDate = arrivalDateFromApplication(this.assessment.application)
 
-    const lateApplicationReason = lateApplicationReasonId
-      ? shortNoticeReasons[lateApplicationReasonId]
-      : 'None supplied'
+    let lateApplicationReason: string
+
+    if (lateApplicationReasonId === 'other') {
+      lateApplicationReason = retrieveOptionalQuestionResponseFromFormArtifact(
```

Comment:
Is it worth providing a fallback here for backwards compatibility?

Reply:
Good call 👍